src/Main.hs

```diff
@@ -5,9 +5,9 @@ module Main
 
 
 --------------------------------------------------------------------------------
-import           Control.Monad            (forM_, unless)
+import           Control.Monad            (forM_, unless, when)
 import qualified Data.ByteString.Char8    as BC8
-import           Data.Monoid              ((<>))
+--import           Data.Monoid              ((<>))
```

Comment:
nitpick: remove me

Reply:
I was unsure of this one - I removed it since it was giving me warnings. But, I guess it's necessary for older version of base?

Reply:
I stubbed this out using the "standard CPP way"